src/Messages/OWSMessageSender.m

```diff
@@ -594,13 +602,17 @@ - (void)deliverMessage:(TSOutgoingMessage *)message
                 ? self.storageManager.localNumber
                 : contactThread.contactIdentifier;
 
+            // If we block a user, don't send 1:1 messages to them. The UI
+            // should prevent this from occurring, but in some edge cases
+            // you might, for example, have a pending outgoing message when
+            // you block them.
             OWSAssert(recipientContactId.length > 0);
             NSArray<NSString *> *blockedPhoneNumbers = _blockingManager.blockedPhoneNumbers;
             if ([blockedPhoneNumbers containsObject:recipientContactId]) {
-                DDLogInfo(@"%@ skipping 1:1 send to blocked contact: %@", self.tag, recipientContactId);
```

Comment:
nit: why remove this log? It's unlikely to be filling the logs up, and if it were I'd almost certainly want to know.